website/react-magma-docs/src/pages/api/accordion.mdx

```diff
@@ -25,7 +25,7 @@ import { LeadParagraph } from '../../components/LeadParagraph';
 
 An accordion is made up of four components: `Accordion`, `AccordionItem`, `AccordionButton` and `AccordionPanel`. The `Accordion` is the wrapper for all of the items. It can contain one or more `AccordionItems`, each of which should contain one `AccordionButton` and one `AccordionPanel`. An `AccordionButton` can be wrapped in an element with the role of heading (such as an h1-h6) to provide semantic structure to the document.
 
-By default, multiple items will can be expanded. The `defaultIndex` prop, which takes an array of zero-based indices, can be used to set panels open by default.
+By default, multiple items are expandable. The `defaultIndex` prop which takes a of zero-based array can be used to set panels open by default.
```

Comment:
This sentence is confusing

Reply:
This isn't on: https://docs-preview-1574--upbeat-sinoussi-f675aa.netlify.app/api/accordion/